ceph_deploy/hosts/centos/install.py

```diff
@@ -4,6 +4,23 @@
 from ceph_deploy.util.paths import gpg
 
 
+def map_components(components):
```

Comment:
sorry, maybe some more context would've helped. Don't make this tiny comment prevent you from moving forward though!